### PR TITLE
Add platform dependencies (oj, jrjackson)

### PR DESCRIPTION
**Pull request by jordansissel**

### CODE REVIEW

File: logstash-event.gemspec [hunk 1]
```diff
@@ -25,4 +26,11 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec"
```
Comment (ph) [inline logstash-event.gemspec]: We need to remove `spec/event.rb` from the list of files. gem build will complains if it doesn't exist.

File: logstash-event.gemspec [hunk 2]
```diff
@@ -25,4 +26,11 @@ Gem::Specification.new do |gem|
...
   gem.add_development_dependency "guard"
   gem.add_development_dependency "guard-rspec"
```
Comment (ph) [inline logstash-event.gemspec]: The guard* gems can be safely removed, we don't have a **Guardfile** in the project.

File: logstash-event.gemspec (hunk 1 above)
Comment (ph) [inline logstash-event.gemspec]: We should use the `lib/logstash/version` to generate the version of this gem.

Comment (ph) [inline logstash-event.gemspec]: We are missing a few dependencies here?

require "cabin"
require "logstash/namespace"
require "logstash/util/accessors"
require "logstash/timestamp"

**Comment by suyograo** [comment]
@jordansissel @jsvd is this a dup of https://github.com/elasticsearch/logstash/pull/2397

**Comment by jordansissel** [comment]
I see how it is.

**Comment by jordansissel** [comment]
:package:

**Pull request closed without merging**